region/crateria/central.json

```diff
@@ -118,6 +118,57 @@
               ]
             }
           ],
+          "leaveWithGMode": [
+            {
+              "leavesWithOverloadedPLMs": true,
+              "leavesWithArtificialMorph": false,
+              "strats": [
+                {
+                  "name": "Overload Scroll PLMs",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "direct",
+                      "artificialMorph": false,
+                      "previouslyOverloadedPLMs": false,
+                      "immobile": false
+                    }}
+                  ],
+                  "note": [
+                    "Enter with direct G-mode, overload the scroll PLMs which are one tile to the left of the bomb blocks, then exit back through the same door."
+                  ]
+                }
+              ]
+            },
+            {
+              "leavesWithOverloadedPLMs": true,
+              "leavesWithArtificialMorph": true,
+              "strats": [
+                {
+                  "name": "Artificial Morph Overload Scroll PLMs",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "Bombs",
+                      "SpringBall"
+                    ]},
```

Comment:
or 7 PBs?

Reply:
Thanks, Bokyubi checked and found this could be done with 4 PBs. Added it now